modules/ROOT/pages/cassandra/cassandra-use-cases.adoc

```diff
@@ -0,0 +1,231 @@
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Host | Enter a host name, or IP address of a Cassandra node.
+|Port | Enter a port number. The default port is 9042.
+|Keyspace | Enter the Cassandra keyspace. A keyspace groups column families.
+|Username | Enter a Cassandra user name if you enabled PasswordAuthenticator in the Cassandra YAML file. If AllowAllAuthenticator is enabled, leave this value blank.
+|Password | Enter the password if you enabled PasswordAuthenticator. Otherwise leave this value blank.
+|===
++
+. Click the Test Connection option to confirm that Mule can connect with the Cassandra instance. If the connection is successful, click OK to save the configuration. Otherwise, review and correct any invalid parameters and retest.
++
+. Run the application. In a browser, use the following URL to enter a query parameter for the keyspace name (ks_name), replication factor (rf), and replication strategy class (rs_class):
++
+`+http://localhost:8081/simple?ks_name=simple_keyspace&rf=3&rs_class=SimpleStrategy+`
+
+=== Create a Cassandra Table
+
+. Create a new Mule Project in Anypoint Studio and fill in the Cassandra credentials in `src/main/resources/mule-app.properties`.
++
+[source,text,linenums]
+----
+config.host=<HOST>
+config.port=<PORT>
+config.keyspace=<KEY_SPACE>
+config.username=<USERNAME>
+config.password=<PASSWORD>
+----
++
+. Drag an HTTP connector onto the canvas and leave the default values for Host and Port and set the path to `/test/createTable`.
++
+. Drag a Transform Message component from the Mule palette to the right (process) side of the flow, select the component, and set the output payload. For example:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/java
+---
+{
+  "columns": payload.columns,
+  "tableName": payload.tableName,
+  "keyspaceName": payload.keyspaceName
+} as Object {
+  class : "org.mule.modules.cassandradb.api.CreateTableInput"
+}
+----
++
+. Drag a Cassandra component for the Create Table operation onto the canvas.
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Host | Enter a host name, or IP address, of a Cassandra node.
+|Port | Enter a port number. The default port is 9042.
+|Keyspace | Enter the Cassandra keyspace. A keyspace groups column families.
+|Username | Enter a Cassandra user name if you enabled PasswordAuthenticator in the Cassandra YAML file. If AllowAllAuthenticator is enabled, leave this value blank.
+|Password | Enter the password if you enabled PasswordAuthenticator; otherwise, leave this value blank.
+|===
++
+. Click the Test Connection option to confirm that Mule can connect with the Cassandra instance. If the client test is successful, click OK to save the configuration. Otherwise, review or correct any invalid parameters and retest.
++
+. Run the Mule app. In Postman, select POST. Select Body > Raw, select the JSON (application/json) MIME type, and enter the following table description using uppercase for data types.
```

Comment:
This is a George or Meera question, but in the Mulesoft Fundamentals Class, they are using the Advanced REST Client for walkthroughs. Postman is very popular though.

Reply:
Can you make sure the team knows the answer to this once you find out?